### DISCUSSION TITLE
Can't find initial values for properties?

### DISCUSSION THREAD

**Discussion by jcmrva** [jbevain/cecil#754] [Q&A] [answered]
Apologies in advance if I'm filing an issue over nothing, but given a property:
```
public string Details { get; set; } = "Disappearing text";
```

How do I get the string value?

I dug into `PropertyDefinition` and the backing fields and didn't find anything.

If it not currently possible, can this be added?

**Comment by jbevain** [comment, marked as answer]
Hi @jcmrva,

I suggest you open the assembly in a tool like `ILSpy`. You'll see that the property of backing field is set in the constructor of your type.

From a metadata perspective, there's nothing tying that value to a field or a property. It's just code.
